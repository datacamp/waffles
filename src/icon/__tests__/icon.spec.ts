import * as icons from '../index';

describe('Icon', () => {
  it('exports every icon', () => {
    expect(icons).toMatchSnapshot();
  });
});
