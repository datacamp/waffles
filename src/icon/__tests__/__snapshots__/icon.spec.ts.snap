// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon exports every icon 1`] = `
Object {
  "AddCircle": [Function],
  "AddCircleInverted": [Function],
  "AddUser": [Function],
  "AppleBrand": [Function],
  "Apply": [Function],
  "Archive": [Function],
  "ArrowDown": [Function],
  "ArrowLeft": [Function],
  "ArrowRight": [Function],
  "ArrowUp": [Function],
  "Assess": [Function],
  "Assignment": [Function],
  "AssignmentInverted": [Function],
  "Attachment": [Function],
  "AttentionCircle": [Function],
  "AttentionCircleInverted": [Function],
  "Back": [Function],
  "BarChart": [Function],
  "Bell": [Function],
  "Book": [Function],
  "Bookmark": [Function],
  "BookmarkInverted": [Function],
  "Branch": [Function],
  "Briefcase": [Function],
  "Building": [Function],
  "Bulb": [Function],
  "Bullet": [Function],
  "Chat": [Function],
  "ChatInverted": [Function],
  "Checkmark": [Function],
  "CheckmarkCircle": [Function],
  "CheckmarkCircleInverted": [Function],
  "ChevronDown": [Function],
  "ChevronLeft": [Function],
  "ChevronRight": [Function],
  "ChevronUp": [Function],
  "Circle": [Function],
  "CircleInverted": [Function],
  "Clock": [Function],
  "ClockInverted": [Function],
  "Code": [Function],
  "Cog": [Function],
  "CogInverted": [Function],
  "Columns": [Function],
  "Command": [Function],
  "Compass": [Function],
  "Copy": [Function],
  "CreditCard": [Function],
  "Cross": [Function],
  "CrossCircle": [Function],
  "CrossCircleInverted": [Function],
  "Crosshair": [Function],
  "DataCampBrand": [Function],
  "Database": [Function],
  "Desktop": [Function],
  "Document": [Function],
  "Dollar": [Function],
  "Download": [Function],
  "Edit": [Function],
  "Education": [Function],
  "Exercise": [Function],
  "ExerciseInverted": [Function],
  "Exit": [Function],
  "Expand": [Function],
  "ExternalLink": [Function],
  "FacebookBrand": [Function],
  "Feed": [Function],
  "Filter": [Function],
  "Flag": [Function],
  "Flash": [Function],
  "FlashInverted": [Function],
  "Flashcard": [Function],
  "Float": [Function],
  "Folder": [Function],
  "Footer": [Function],
  "Forbidden": [Function],
  "Forward": [Function],
  "FullScreen": [Function],
  "GithubBrand": [Function],
  "Globe": [Function],
  "GoogleBrand": [Function],
  "GoogleColoredBrand": [Function],
  "GooglePlayBrand": [Function],
  "Group": [Function],
  "GroupInverted": [Function],
  "Header": [Function],
  "Health": [Function],
  "HealthInverted": [Function],
  "Heart": [Function],
  "HeartInverted": [Function],
  "Help": [Function],
  "HelpCircle": [Function],
  "HelpCircleInverted": [Function],
  "Hidden": [Function],
  "HiddenInverted": [Function],
  "Home": [Function],
  "InfoCircle": [Function],
  "InfoCircleInverted": [Function],
  "InstagramBrand": [Function],
  "Interface": [Function],
  "Introduction": [Function],
  "KaggleBrand": [Function],
  "Keyboard": [Function],
  "Lab": [Function],
  "LeanpubBrand": [Function],
  "Learn": [Function],
  "LineChart": [Function],
  "Link": [Function],
  "LinkedInBrand": [Function],
  "List": [Function],
  "Locked": [Function],
  "LockedInverted": [Function],
  "Mail": [Function],
  "Markdown": [Function],
  "Matrix": [Function],
  "Medal": [Function],
  "Menu": [Function],
  "Microphone": [Function],
  "MicrophoneInverted": [Function],
  "Mobile": [Function],
  "Moon": [Function],
  "MoonInverted": [Function],
  "More": [Function],
  "Mute": [Function],
  "MuteInverted": [Function],
  "NineDots": [Function],
  "Pause": [Function],
  "PaypalBrand": [Function],
  "PaypalColoredBrand": [Function],
  "Phone": [Function],
  "PieChart": [Function],
  "Pin": [Function],
  "PinInverted": [Function],
  "PlatformSwitch": [Function],
  "Play": [Function],
  "PlayInverted": [Function],
  "Practice": [Function],
  "PythonBrand": [Function],
  "RBrand": [Function],
  "Random": [Function],
  "Redo": [Function],
  "Remove": [Function],
  "RemoveInverted": [Function],
  "Reply": [Function],
  "Rewind": [Function],
  "RewindInverted": [Function],
  "Rocket": [Function],
  "Rows": [Function],
  "Save": [Function],
  "ScalaBrand": [Function],
  "Search": [Function],
  "Share": [Function],
  "Skip": [Function],
  "SkipInverted": [Function],
  "SlackBrand": [Function],
  "SlackColoredBrand": [Function],
  "Sound": [Function],
  "SoundInverted": [Function],
  "Sparkles": [Function],
  "SparklesInverted": [Function],
  "Star": [Function],
  "StarInverted": [Function],
  "Stop": [Function],
  "StopInverted": [Function],
  "Sun": [Function],
  "SunInverted": [Function],
  "Tab": [Function],
  "Tag": [Function],
  "Team": [Function],
  "Terminal": [Function],
  "TerminalInverted": [Function],
  "Text": [Function],
  "Trash": [Function],
  "TrashInverted": [Function],
  "TrimBegin": [Function],
  "TrimEnd": [Function],
  "Trophy": [Function],
  "TwitterBrand": [Function],
  "Undo": [Function],
  "Unlink": [Function],
  "Unlocked": [Function],
  "UnlockedInverted": [Function],
  "Upload": [Function],
  "Upvote": [Function],
  "UpvoteInverted": [Function],
  "User": [Function],
  "UserInverted": [Function],
  "Video": [Function],
  "VideoCamera": [Function],
  "VideoCameraInverted": [Function],
  "VideoInverted": [Function],
  "Visible": [Function],
  "VisibleInverted": [Function],
  "Waypoint": [Function],
  "YoutubeBrand": [Function],
}
`;
